.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Had to upgrade cake as current appveyor win images dont have required .netcore version (2.x.x)
https://ci.appveyor.com/project/NSubstitute/nsubstitute-analyzers/builds/48719348/job/d4iwb70xjucs5cag

Reply:
@tpodolak We've migrated the NSub repo to GitHub actions recently https://github.com/nsubstitute/NSubstitute/blob/main/.github/workflows/build_and_test.yml
I can help with that if you want.

Reply:
Thanks @alexandrnikitin. I've added task to migrate analyzers as well https://github.com/nsubstitute/NSubstitute.Analyzers/issues/216